Library/ViewModels/LoginToutViewModelTests.swift

```diff
@@ -585,4 +590,26 @@ final class LoginToutViewModelTests: TestCase {
 
     self.dismissViewController.assertValueCount(1)
   }
+
+  @available(iOS 13, *)
+  func testDidSignInWithApple() {
+    let response = SignInWithAppleEnvelope.template
+
+    withEnvironment(apiService: MockService(signInWithAppleResponse: response)) {
```

Comment:
Can you also add a test for the case where the `signInWithApple` request fails?

Reply:
The test for when the `signInWithApple` fails was added on the second part of this PR, where we also handle errors.